### PR TITLE
Prepare 2.0.3

### PR DESCRIPTION
**Pull request by smorimoto**
#### Changes

- #64

### CODE REVIEW

**Comment by smorimoto** [comment]
This PR also addresses more than what #66 does.

File: biome.json [hunk 1]
```diff
@@ -1,8 +1,5 @@
 {
   "$schema": "node_modules/@biomejs/biome/configuration_schema.json",
-  "files": {
-    "ignoreUnknown": true
-  },
```
Comment (smorimoto) [inline biome.json:5]: This was needed to work around a bug in Biome, but that bug has now been fixed and is no longer needed!

**Comment by smorimoto** [comment]
Most of the changes in the README are whitespace changes, and I recommend checking the rendered markdown with visual diff mode to see the actual diff.

**Review by febuiles — APPROVED** [review]

**Comment by febuiles** [comment]
Merged with 2.0.3: https://github.com/github/dependency-submission-toolkit/releases/tag/v2.0.3 -- thank you!

**Comment by smorimoto** [comment]
Thanks a lot!

**Pull request merged**